schema.json

```
@@ -0,0 +1,263 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
```

Use latest spec instead: https://json-schema.org/specification
```suggestion
	"$schema": "https://json-schema.org/draft/2020-12/schema",
```